<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogProfessorDegree</class>
 <widget class="QDialog" name="dialogProfessorDegree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1207</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gbProDegreeContent">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>591</width>
     <height>461</height>
    </rect>
   </property>
   <property name="title">
    <string>Contents</string>
   </property>
   <widget class="QTableView" name="tvProfessorDegree">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>551</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_7" native="true">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>340</y>
     <width>281</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <widget class="QRadioButton" name="rdAddDegree">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add professor degree</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdbUpdateDegree">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Update professor degree</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rdDeleteDegree">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete professor degree</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbProDegreeEditData">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>Edit data</string>
   </property>
   <widget class="QLabel" name="lbDegreeCode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Degree code:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDegreeName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Degree name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbDegreeSalary">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Salary index:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDegreeCode">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDegreeName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dsbSalaryIndex">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>211</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushBtnDegree">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>370</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Dynamic Button</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
